hwcomposer/hwcomposer_backend_v20.cpp

```diff
@@ -223,6 +228,7 @@ HwComposerBackend_v20::HwComposerBackend_v20(hw_module_t *hwc_module, void *libm
     procs->on_hotplug_received = hwc2_callback_hotplug;
     procs->on_refresh_received = hwc2_callback_refresh;
     procs->backend = this;
+    procs->primaryDisplayId = 0;
```

Comment:
For sake of clarity maybe worth adding a note that initialization steps within the HwComposerBackend_v20 constructor do not alter the primaryDisplayId value. This should be self-evident for libhybris developers as it is HWC2EventListener pointer that's been passed around and hwc2_compat_device_get_display_by_id passing is by value.

Reply:
Is it ok now?